include/alpaka/core/ApiHipRt.hpp

```diff
@@ -11,17 +11,17 @@
 
 #ifdef ALPAKA_ACC_GPU_HIP_ENABLED
 
-#    include <alpaka/core/BoostPredef.hpp>
-
+#    include <boost/predef.h>
 #    include <hip/hip_runtime_api.h>
+#    include <hip/hip_version.h>
 
 namespace alpaka
 {
     struct ApiHipRt
     {
         // Names
         static constexpr char name[] = "Hip";
-        static constexpr auto version = BOOST_LANG_HIP;
+        static constexpr auto version = BOOST_VERSION_NUMBER(HIP_VERSION_MAJOR, HIP_VERSION_MINOR, 0);
```

Comment:
Why do you switch from BOOST_VERSION to the version format ROCM is shipping? Is this necessary?

I am asking because normally we use always BOOST_VERSIONS in alpaka and now we need to take care to use for HIP the HIP internal format.

Reply:
hi @psychocoderHPC, I'm not sure I follow, are you asking why I used the BOOST_VERSION macro explicitly, or why I didn't include the third field in it ?

Reply:
I misread this line I thought you do not use `BOOST_VERSION` anymore but you use it. I was confused by the usage later, I will mark the line.